src/main/scala/ch/epfl/bluebrain/nexus/kg/directives/ProjectDirectives.scala

```diff
@@ -16,14 +17,22 @@ object ProjectDirectives {
   /**
     * Fetches project configuration from nexus admin
```

Comment:
Fetches project configuration from the cache if possible, from nexus admin otherwise.